R/rd-html.R

```diff
@@ -185,7 +207,9 @@ as_html.tag_link <- function(x, ...) {
 
 #' @export
 as_html.tag_linkS4class <- function(x, ...) {
-  stopifnot(length(x) == 1)
+  if ( length(x) != 1 ) {
```

Comment:
Can you please remove the extra spaces inside the parens? (https://style.tidyverse.org/syntax.html#parentheses)